modules/conversion-center/base/widget-contact-button-base.php

```diff
@@ -1429,12 +1432,13 @@ protected function add_style_top_bar_section(): void {
 					[
 						'label' => esc_html__( 'Close Button', 'elementor' ),
 						'type' => Controls_Manager::HEADING,
-						'separator' => false,
+						'separator' => $close_and_background_partial_divider,
```

Comment:
@willhowat can we add this to the config instead? So you would add it at the top of this file in the `style` -> `top_bar_section` as a variable like `has_close_and_background_separator` and set it to `false` by default
and you would access it like this `$config['style']['top_bar_section']['has_close_and_background_separator']` from the variant that needs it

Reply:
Hi @mserino this configuration is as far as I'm aware required across all variations.
What we're seeing here and may not be apparent just through the changes view, is a local variable used to control where this border is applied as it always has to be applied to the first control in this group, but visibility of some controls is optional. So we set this to show by default and then null it out as we render the controls.

If at some point this needs to move into configuration we can add a config item that sets the initial state, but at this time it does not vary across variants, unless there is one I've not seen

Reply:
makes sense, thanks!